### ISSUE TITLE
Support class_weight and sample_weight in tfpark KerasModel.fit

### ISSUE THREAD

**Issue by hkvision** [intel-analytics/analytics-zoo#3309]
https://groups.google.com/g/bigdl-user-group/c/08SUl5pjDA0/m/R4sFE5NuBAAJ
cc @yangw1234

**Comment by hkvision** [comment]
Seems class_weight is the same as the option in ClassNLLCriterion, but sample_weight can assign weight to each input sample, which is not supported.

**Comment by yangw1234** [comment]
we will recommend users to use tf2 estimator, which already supports this feature.

**Comment by adarshpatankar28** [comment]
Hey @yangw1234 , tf2 estimator supports sample_weight or class weight feature ?
 Also can you please share link/documentation/example code of tf2 estimator using this feature.

**Comment by yangw1234** [comment]
> Hey @yangw1234 , tf2 estimator supports sample_weight or class weight feature ?
> Also can you please share link/documentation/example code of tf2 estimator using this feature.

hi @adarshpatankar28 , you can refer to this quick start https://analytics-zoo.readthedocs.io/en/latest/doc/Orca/QuickStart/orca-tf2keras-quickstart.html.

You can specify `class_weight` in the estimator `fit` method. For `sample_weights`, you can make your `data_creator` return a tf.data.Dataset, which outputs `(feature, label, sample_weights)` tuple. It is basically the same as `tf.keras`.

**Comment by helenlly** [comment]
@adarshpatankar28 any more questions on the issue? if no, we may close it.
